<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultChild</class>
 <widget class="QMdiSubWindow" name="ResultChild">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QProgressBar" name="prog1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>871</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
     border: 2px solid grey;
     border-radius: 5px;
     background-color: transparent;
 }

 QProgressBar::chunk {
     background-color: #FF0000;
     width: 20px;
 }</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ans1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>871</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid grey;
border-radius: 5px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Answer1</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="prog2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>871</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
     border: 2px solid grey;
     border-radius: 5px;
     background-color: transparent;
 }

 QProgressBar::chunk {
     background-color: #FF0000;
     width: 20px;
 }</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ans2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>871</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid grey;
border-radius: 5px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Answer2</string>
   </property>
  </widget>
  <widget class="QLabel" name="ans3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>871</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid grey;
border-radius: 5px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Answer3</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="prog3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>600</y>
     <width>871</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
     border: 2px solid grey;
     border-radius: 5px;
     background-color: transparent;
 }

 QProgressBar::chunk {
     background-color: #FF0000;
     width: 20px;
 }</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>1280</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Question&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gotomenu">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>600</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Return to Menu</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QMdiSubWindow</class>
   <extends>QWidget</extends>
   <header>qmdisubwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
